extensions/amp-story/1.0/amp-story-open-page-attachment.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * Copyright 2021 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @fileoverview Helper for amp-story rendering of page-attachment UI.
+ */
+import {htmlFor} from '../../../src/static-template';
+import {getLocalizationService} from './amp-story-localization-service';
+import {LocalizedStringId} from '../../../src/localized-strings';
+
+/**
+ * @param {!Element} element
+ * @return {!Element}
+ */
+export const buildOpenAttachmentElement = (element) =>
+htmlFor(element)`
+    <a class="
+        i-amphtml-story-page-open-attachment i-amphtml-story-system-reset"
+        role="button">
+      <span class="i-amphtml-story-page-open-attachment-icon">
+        <span class="i-amphtml-story-page-open-attachment-bar-left"></span>
+        <span class="i-amphtml-story-page-open-attachment-bar-right"></span>
+      </span>
+      <span class="i-amphtml-story-page-open-attachment-label"></span>
+    </a>`;
+
+/**
+ * Renders the open attachment UI affordance.
+ */
+export function renderPageAttachmentUI(el, attachmentEl, openAttachmentEl) {
```

Comment:
I think it would make sense to build the template in this function and return it instead of passing it in as an argument and operating on it. It would help encapsulate the logic further.

Reply:
Yeah this makes sense, especially since there'll be new elements with the new UI.